internal/gatewayapi/route.go

```diff
@@ -120,17 +121,18 @@ func (t *Translator) processHTTPRouteParentRefs(httpRoute *HTTPRouteContext, res
 		// any conditions that come out of it have to go on each RouteParentStatus,
 		// not on the Route as a whole.
 		routeRoutes, err := t.processHTTPRouteRules(httpRoute, parentRef, resources)
+		// TODO: zhaohuabing: according to the gateway api, the RouteConditionPartiallyInvalid condition should be set
```

Comment:
can we make these changes in a separate PR ?

Reply:
Planning to implement "RouteConditionPartiallyInvalid" in a separate PR. The other changes are made during adding the Dyanamic Resolver to the route translation.